atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/todo/TodoClient.java

```diff
@@ -54,7 +54,7 @@ public void addTodo(Todo todo) {
                     TodoSchema.todoTable(), RangeRequest.all());
             ImmutableList.Builder<RowResult<byte[]>> rowResults = ImmutableList.builder();
 
-            rowResultBatchingVisitable.batchAccept(1000, items -> {
+            rowResultBatchingVisitable.batchAccept(10, items -> {
```

Comment:
parameters like this make the tests brittle; would be better to have all batch sizes explicitly defined by the tests (another reason to use the txn manager directly)

Reply:
They are gone now.